# 客户端请求时发送的两个数字
string req_detector
string req_object
int32 munber
---
# 服务器响应发送的数据
bool success
string msg